frontend/src/components/limitRange/Details.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, Typography } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router';
+import { LimitRange } from '../../lib/k8s/limitRange';
+import { DetailsGrid, MetadataDictGrid } from '../common';
+
+export function LimitRangeDetails() {
+  const { t } = useTranslation('glossary');
+  const { namespace, name } = useParams<{ namespace: string; name: string }>();
+  return (
+    <DetailsGrid
+      resourceType={LimitRange}
+      name={name}
+      namespace={namespace}
+      withEvents
+      extraInfo={item =>
+        item && [
+          {
+            name: t('Container Limits'),
```

Comment:
This should be in the resource namespace.

Reply:
fixed